src/platforms/mirserver/windowmanagementpolicy.cpp

```diff
@@ -74,7 +74,10 @@ miral::WindowSpecification WindowManagementPolicy::place_new_window(
 
 void WindowManagementPolicy::handle_window_ready(miral::WindowInfo &windowInfo)
 {
-    CanonicalWindowManagerPolicy::handle_window_ready(windowInfo);
+    // Lets not invoke this as it will result in the window getting automaticly activated
+    // regardless if it's placeholder in unity8 is focused or not. We should let
+    // unity8 deside when to focus a new window.
+    // CanonicalWindowManagerPolicy::handle_window_ready(windowInfo);
```

Comment:
Please just remove code rather than commenting it out. The comment can be the commit message, and the code will remain in older revisions in git, so there's no reason to keep it there but commented out.